docs/src/main/sphinx/connector/hive.rst

```diff
@@ -398,6 +398,16 @@ Property Name                                      Description
                                                    Set to ``false`` to disable statistics. Disabling statistics
                                                    means that :doc:`/optimizer/cost-based-optimizations` can
                                                    not make smart decisions about the query plan.
+
+``hive.managed-table-auto-purge-default``          Sets default value for managed table property auto_purge`.    ``false``
+                                                   You can use this property to set auto_purge at catalog level
+                                                   and skip setting auto_purge table property during managed
+                                                   table creation.
+                                                   auto_purge managed table property when set to true overrides
+                                                   this catalog property and skips trash for the managed table
+                                                   deletion.
+                                                   See the :ref:`hive_table_properties` for more information
+                                                   on auto_purge.
```

Comment:
I would also mention somewhere that auto purge only applies to a Hive metastore and not other other implementations such as Glue.